ingenialink/utils/mcb.py

```diff
@@ -139,7 +139,7 @@ def read_mcb_data(cls, expected_address, frame):
         if expected_address != recv_add:
             raise ILWrongRegisterError(
                 f"Received address: {hex(recv_add)} does "
-                f"not match expected address: "
+                "not match expected address: "
```

Comment:
I know there is no any variable in this line, but I think it's a little weird that this line doesn't have an f-string, when the other lines that belong to this string have it.
What do you think?

Reply:
I think you should only use an f-string when needed, in this case, it is not.